recording-addons-watson-csharp/Web.config

```diff
@@ -1,53 +1,52 @@
-﻿<?xml version="1.0" encoding="utf-8"?>
+﻿<?xml version="1.0"?>
 <!--
   For more information on how to configure your ASP.NET application, please visit
   https://go.microsoft.com/fwlink/?LinkId=301880
   -->
 <configuration>
-  <appSettings>
-    <add key="webpages:Version" value="3.0.0.0" />
-    <add key="webpages:Enabled" value="false" />
-    <add key="TwilioAccountSid" value="ACxxx" />
-    <add key="TwilioAuthToken" value="xxx" />
-    <add key="ClientValidationEnabled" value="true" />
-    <add key="UnobtrusiveJavaScriptEnabled" value="true" />
+  <appSettings file="Local.config">
+    <add key="webpages:Version" value="3.0.0.0"/>
+    <add key="webpages:Enabled" value="false"/>
+    <!-- Insert your values here, or put" them in Local.config -->
+    <add key="TwilioAccountSid" value="ACxxx"/>
+    <add key="TwilioAuthToken" value="xxx"/>

```

Comment:
Not needed anymore

Reply:
They are here just as reference and the comment explains it a bit.